### PR TITLE
solution

### PR DESCRIPTION
**Pull request by NikitaKarbazin**
[DEMO LINK](https://nikitakarbazin.github.io/react_list-of-goods/)

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1 +1,20 @@
...
+  justify-content: center;
+}
+
+li {
+  font-size: 30px;
+}
+
+button {
+  margin: 100px;
+}
```
Comment (stanislav-topikha) [inline src/App.css:20]: Not the best idea to style components with tags

File: src/App.tsx [hunk 2]
```diff
@@ -14,11 +14,124 @@ const goodsFromServer: string[] = [
...
+    return {
+      id: index,
+      name: good,
+    };
+  });
+};
+
+type State = {
+  listOfGoods: ComparedGood[],
+  hasRightEntrance: boolean,
```
Comment (stanislav-topikha) [inline src/App.tsx:33]: very confusing name, maybe isListShowed or smth else

**Review by stanislav-topikha — APPROVED** [review]
Left a few comments, overall good

File: src/App.css (hunk 1 above)
Comment (mykola-koval) [inline src/App.css:20]: not really a good practice to style tags, style using classes

File: src/App.tsx (hunk 2 above)
Comment (mykola-koval) [inline src/App.tsx:33]: i would change the name too

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**